package.json

```diff
@@ -136,45 +136,45 @@
     "@types/csurf": "^1.11.2",
     "@types/express-session": "^1.17.7",
     "@types/http-errors": "^2.0.1",
-    "@types/jest": "^29.5.1",
-    "@types/jsonwebtoken": "^9.0.1",
-    "@types/node": "^18.16.0",
-    "@types/nunjucks": "^3.2.2",
+    "@types/jest": "^29.5.2",
+    "@types/jsonwebtoken": "^9.0.2",
+    "@types/node": "^20.3.2",
```

Comment:
We're still using node 18.x so better to stick with those types

Reply:
ah nice spot ill put that back to what it was